### PR TITLE
Integrate feature toggles into CTA Widget, specifically for VAOS

### PR DESCRIPTION
**Pull request by narin**
## Description
This updates the CTA widget to switch between two different online scheduling views, depending on a feature flag. Most of the rearranging in the CTAWidget component is too avoid MHV related checks when the VAOS feature flag is on.

## Testing done
Local testing

## Screenshots
Screen Shot 2020-02-04 at 1 43 07 PM

## Acceptance criteria
- [ ] When feature flag is on, new content is shown

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jbalboni — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]
The LGTM, but it's hard to follow some of the CTA logic.

File: src/applications/post-911-gib-status/tests/post-911-gib-status-helpers.js [hunk 1]
```diff
@@ -99,6 +99,16 @@ function initApplicationMock(token) {
...
+
+  mock(token, {
+    path: '/v0/feature_toggles',
+    verb: 'get',
+    value: {
+      data: {
+        features: [],
+      },
+    },
+  });
```
Comment (ncksllvn) [inline src/applications/post-911-gib-status/tests/post-911-gib-status-helpers.js]: I really love seeing feature toggles used like this. It sets up the E2E test really nicely for a smooth migration.

**Review by bkjohnson — APPROVED** [review]

**Review by bkjohnson — COMMENTED** [review]
I didn't mean to re-request myself for review - I just was a bit wary of some of the site-wide changes but it seems ok.

**Comment by jbalboni** [comment]
@bkjohnson Thanks. If there's something in here you want to go over or have us explain, let me know. I asked about making these changes to the widget in the vsp tools channel a couple weeks ago, but didn't get much response either way.

**Comment by bkjohnson** [comment]
The big question I have: 
 
Is the `loading` feature toggle not _actually_ a feature toggle, but an indicator of whether the other feature toggles (like `vaOnlineScheduling`) have been loaded from the backend yet?

**Comment by jbalboni** [comment]
@bkjohnson Yes, that's correct. `loading` is a loading status, not a toggle, even though it's in the feature toggles state slice.
